Ioss: missing header

It appears there is a missing header included in an Ioss file that is causing Albany build failures:

```
/projects/albany/nightlyCDashWeaver/build/TrilinosInstall/include/Ioss_Field.h:12:10: fatal error: fmt/ostream.h: No such file or directory
```

https://sems-cdash-son.sandia.gov/cdash/viewBuildError.php?buildid=91235

--- a/packages/seacas/libraries/ioss/src/Ioss_Field.h
+++ b/packages/seacas/libraries/ioss/src/Ioss_Field.h
@@ -8,9 +8,6 @@
 
 #include "Ioss_CodeTypes.h"
 #include <cstddef> // for size_t
-#if !defined BUILT_IN_SIERRA
-#include <fmt/ostream.h>
-#endif
 #include <stdint.h>
 #include <string> // for string
 #include <vector> // for vector
@@ -247,13 +244,3 @@ namespace Ioss {
   };
   IOSS_EXPORT std::ostream &operator<<(std::ostream &os, const Field &fld);
 } // namespace Ioss
-
-#if !defined BUILT_IN_SIERRA
-#if FMT_VERSION >= 90000
-namespace fmt {
-  template <> struct formatter<Ioss::Field> : ostream_formatter
-  {
-  };
-} // namespace fmt
-#endif
-#endif

--- a/packages/seacas/libraries/ioss/src/Ioss_Utils.C
+++ b/packages/seacas/libraries/ioss/src/Ioss_Utils.C
@@ -1369,7 +1369,7 @@ void Ioss::Utils::info_fields(const Ioss::GroupingEntity *ige, Ioss::Field::Role
   for (const auto &field_name : fields) {
     if (detail) {
       const auto &field_ref = ige->get_fieldref(field_name);
-      fmt::print("{}{}", field_ref, suffix);
+      std::cout << field_ref << suffix;
     }
     else {
       const Ioss::VariableType *var_type   = ige->get_field(field_name).raw_storage();

